src/admin-views/editor/fieldset/advanced.php

```diff
@@ -52,6 +52,24 @@
 			class="ticket_field ticket_form_right"
 			id="ticket_description"
 		><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+		<div class="input_block">
+			<label class="tribe_soft_note">
+				<input
+						type="checkbox"
+						id="tribe_tickets_show_type"
+						name="ticket_show_type"
+						value="1"
+						class="ticket_field ticket_form_left"
+						<?php checked( true, $ticket ? $ticket->show_type : true ); ?>
+				>
+				<?php
+				echo esc_html( sprintf(
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: b02f0dde6a8e976b3c67beb876e38abb</sub>

Reply:
Let's address this.